src/component/base/DraftEditor.css

```diff
@@ -13,9 +13,9 @@
   text-align: initial;
 }
 
-.public/DraftEditor/content[contenteditable="true"] {
+/* .public/DraftEditor/content[contenteditable="true"] {
```

Comment:
This change _will_ leak through to other browsers. Do we know what this was for?

Reply:
Oh 🤦‍♂ Good catch!

This prevents iOS from showing its popup menu.  We should check the behavior of that against ipad and iphone before merging.  
We do need this one way or another though since including it means that chrome on android will not emit ranges in beforeInput.  

It may be something we need to do in JS…

Reply:
we also have the code duplicated in editor in case we want to conditionally add it there.

this is the only piece of code i'm not sure of as far as interacting with current desktop/ios stuff

Reply:
@adamsanderson do you have any insight? or should I just test this on safari/ios ?

Reply:
would you all (@fraziermork @adamsanderson) rather this be applied in editor or here in draft ? I can do a quick css-in-js fix, but wondering where best to put it

Reply:
It was in the base draft for a reason probably, so I'd do it in editor I guess 🤷‍♂